readme.md

```diff
@@ -297,6 +334,13 @@ To summarize - if you're 100% sure that `php artisan basset:cache` is pulling al
 
 If you require customized behavior after each asset is cached, you can set up a listener for the `BassetCachedEvent` in your `EventServiceProvider`. This event will be triggered each time an asset is cached.
 
+## Upgrading from v1 to v2
+
+- Run `php artisan basset:install --git-ignore` to refresh composer hooks and optionally ignore the public cache folder.
```

Comment:
These 4 would have worked better as "step 1", step 2, step 3. But ok.

Very nicely phrased. Love the "warmed up the cache"